javascript/javascript1/week1/script.js

```diff
@@ -0,0 +1,103 @@
+// FreeCodeCamp user: https://www.freecodecamp.org/catalina-tech
+
+
+
+// Age-ify (A future age calculator)
+
+let yearOfBirth;
+yearOfBirth = 1988;
+console.log(typeof yearOfBirth);
+
+let yearFuture;
+yearFuture = 2028;
+console.log(typeof yearFuture);
+
+let age = yearFuture - yearOfBirth;
+console.log("You will be" + " " + age + " " + "in" + " " + yearFuture); 
+
+
+
+
+//Goodboy-Oldboy (A dog age calculator)
+
+let dogYearOfBirth;
+dogYearOfBirth = 2017;
+
+let dogYearFuture;
+dogYearFuture = 2027;
+
+//Human years
+let dogYear = dogYearFuture - dogYearOfBirth;
+
+//Dog Years
+let dogYearOld = dogYear*7;
+
+let shouldShowResultInDogYears = false;
+
+if (shouldShowResultInDogYears) {
+    console.log("Your dog will be" + " " + dogYearOld + " " + "dog years old in " + dogYearFuture);    
+} else {
+    console.log("Your dog will be" + " " + dogYear + " " + "human years in " + dogYearFuture);  
+};
+
+
+
+
+//Housey pricey (A house price estimator)
+
+let volumeInMeters;
+let gardenSizeinM2;
+let housePrice;
+housePrice = (volumeInMeters * 2.5 * 1000) + (gardenSizeinM2 * 300);
```

Comment:
On this line the `housePrice` will be `NaN`. (You can try logging the variable: `console.log(housePrice)`). It is happening because the `volumeInMeters` and `gardenSizeinM2` variables don't have any starting values, so javascript doesn't know what value should it use when trying to calculate line 51.
```
housePrice = (volumeInMeters * 2.5 * 1000) + (gardenSizeinM2 * 300);
// This line becomes this:
// housePrice = (????? * 2.5 * 1000) + (????? * 300);
// so JS can't calculate it
// Technically speaking the ????? equals undefined if you want to look deeper into it.
```